### ISSUE TITLE
Repeated consonants in multi-word proquints are hard to enunciate

### ISSUE THREAD

**Issue by DanielHeath** [dsw/proquint#6]
From https://arxiv.org/html/0901.4016 : 

    127.0.0.1       lusab-babad
    63.84.220.193   gutih-tugad
    63.118.7.35     gutuk-bisog
    ...
    216.68.232.21   todah-vobij
    198.81.129.136  sinid-makam
    12.110.110.204  budov-kuras

The first entry in this list is much more difficult to say than the others because the `b` consonant is repeated. 

How badly would the bitrate be affected by selecting the first consonant from a shortened list?

**Comment by dsw** [comment]
The design is public so you can check this yourself.  Frankly I think it is
a fools errand.  How many more properties do you want?

**Comment by DanielHeath** [comment]
I apologize; clearly I've bothered you, which couldn't be further from my intent. 

This is such a cool piece of work that it's easy to get excited about further developments & seeing how far it could be taken.

I think some simple modelling of which adjacent consonants are difficult to pronounce might result in a small loss of bitrate for much clearer enounciation.

**Comment by dsw** [comment]
You have not really annoyed me, it's just that one of the properties of the
system is that the encoding is very simply reductionistic: (1) you chop the
bits into pieces and (2) map them to letters.  You get both (a) a decent
bit rate and (b) something guaranteed to be pronounceable.  I find the
combination of these properties to be rather remarkable.

You could wish for more, but the encoding needed to get it will immediately
be much more complex to the point where I think you will end up with
basically an entirely new encoding.  That will be an entirely new project,
rather than a version 1.1 of this one.  Let me know if you manage to pull
that off.

Daniel
